src/widgets/DiagramWidget.tsx

```diff
@@ -287,11 +287,9 @@ export class DiagramWidget extends React.Component<DiagramProps, DiagramState> {
 			if (this.props.allowCanvasTranslation) {
 				diagramModel.setOffset(
 					this.state.action.initialOffsetX +
-						(event.clientX - this.state.action.mouseX) /
-							(diagramModel.getZoomLevel() / 100),
```

Comment:
Im almost certain that this is whats causing the issue, is it possible to move this down to the zoom logic as well?

Reply:
never mind xD